controller/composite/metacontroller.go

```diff
@@ -168,10 +168,21 @@ func (mc *Metacontroller) sync(key string) error {
 		}
 		return nil
 	}
+
 	if err != nil {
 		mc.eventRecorder.Eventf(cc, v1.EventTypeNormal, events.ReasonSyncError, "[%s] Sync error - %s", cc.Name, err)
 		return err
 	}
+	parentClient, err := mc.dynClient.Resource(cc.Spec.ParentResource.APIVersion, cc.Spec.ParentResource.Resource)
+	if err != nil {
+		return err
+	}
+	if hasStatusSubresource := parentClient.APIResource.HasSubresource("status"); !hasStatusSubresource {
+		mc.eventRecorder.Eventf(cc, v1.EventTypeWarning, events.ReasonSyncError, "[%s] Sync error - ignoring, parent resource %s does not have subresource 'Status' enabled", cc.Name, parentClient.GroupVersionKind())
+		klog.InfoS("Ignoring CompositeController", "name", name, "reason", fmt.Sprintf("subresource 'Status' not enabled for %s", parentClient.GroupVersionKind()))
```

Comment:
We should put this under some log level e.g. klog.V(2).InfoS

Reply:
actually the idea is to make this message verbose, as it is quite important (see related bug)